jenkins/ompi/ompi_test.sh

```diff
@@ -1,105 +1,98 @@
-jenkins_test_threads=${jenkins_test_threads:="yes"}
-jenkins_test_cov=${jenkins_test_cov:="no"}
-jenkins_test_known_issues=${jenkins_test_known_issues:="no"}
-jenkins_test_all=${jenkins_test_all:="no"}
-jenkins_test_debug=${jenkins_test_debug:="no"}
-jenkins_test_slurm=${jenkins_test_slurm:="no"}
-jenkins_test_comments=${jenkins_test_comments:="no"}
-jenkins_test_ucx=${jenkins_test_ucx:="yes"}
-#jenkins_test_vg=${jenkins_test_vg:="yes"}
-jenkins_test_vg="no"
-#jenkins_test_xrc=${jenkins_test_xrc:="yes"}
-jenkins_test_xrc="no"
-jenkins_test_use_ucx_branch=${jenkins_test_use_ucx_branch:="no"}
-jenkins_test_ucx_branch=${jenkins_test_ucx_branch:="master"}
-jenkins_test_hcoll="no"
+cd "$WORKSPACE"
+export PATH="/hpc/local/bin:/usr/local/bin:/bin:/usr/bin:/usr/sbin:${PATH}"
+
+help_txt_list="${help_txt_list:="oshmem ompi/mca/coll/hcoll ompi/mca/pml/ucx ompi/mca/spml/ucx"}"
+hca_port="${hca_port:=1}"
+ci_test_build=${ci_test_build:="yes"}
+ci_test_examples=${ci_test_examples:="yes"}
+ci_test_oshmem=${ci_test_oshmem:="yes"}
+ci_test_check=${ci_test_check:="yes"}
+ci_test_src_rpm=${ci_test_src_rpm:="no"}
+ci_test_help_txt=${ci_test_help_txt:="no"}
+ci_test_threads=${ci_test_threads:="yes"}
+ci_test_cov=${ci_test_cov:="no"}
+ci_test_known_issues=${ci_test_known_issues:="no"}
+ci_test_debug=${ci_test_debug:="no"}
+ci_test_slurm=${ci_test_slurm:="no"}
+ci_test_comments=${ci_test_comments:="no"}
+ci_test_ucx=${ci_test_ucx:="yes"}
+ci_test_vg="no"
+ci_test_xrc="no"
+ci_test_use_ucx_branch=${ci_test_use_ucx_branch:="no"}
+ci_test_ucx_branch=${ci_test_ucx_branch:="master"}
+ci_test_hcoll="yes"
 
 # Ensure that we will cleanup all temp files
 # even if the application will fail and won't
 # do that itself
-jenkins_session_base=`mktemp -d`
-function jenkins_cleanup {
-  EXIT_CODE=$?
-  echo "Script exited with code = ${EXIT_CODE}"
-  rm -rf "$jenkins_session_base"
-  echo "rm -rf ... returned $?"
-  if [ "${EXIT_CODE}" -eq 0 ]; then
-    echo "PASS"
-  else
-    echo "FAIL"
-  fi
-  exit ${EXIT_CODE}
+ci_session_base=$(mktemp -d)
+
+function ci_cleanup {
+    EXIT_CODE=$?
+    echo "Script exited with code = ${EXIT_CODE}"
+    rm -rf "${ci_session_base}"
+    echo "rm -rf ... returned $?"
+
+    if [ "${EXIT_CODE}" -eq 0 ]
+    then
+        echo "PASS"
+    else
+        echo "FAIL"
+    fi
+
+    exit ${EXIT_CODE}
 }
-trap jenkins_cleanup EXIT
-export OMPI_MCA_orte_tmpdir_base=$jenkins_session_base
 
-if [ -n "$EXECUTOR_NUMBER" ]; then
-    AFFINITY_GLOB="taskset -c $(( 2 * EXECUTOR_NUMBER ))","$(( 2 * EXECUTOR_NUMBER + 1))"
+trap ci_cleanup EXIT
+
+export OMPI_MCA_orte_tmpdir_base="${ci_session_base}"
+
+EXECUTOR_NUMBER=${EXECUTOR_NUMBER:="none"}
+
+if [ "${EXECUTOR_NUMBER}" != "none" ]
+then
+    AFFINITY_GLOB="taskset -c $(( 2 * EXECUTOR_NUMBER )),$(( 2 * EXECUTOR_NUMBER + 1))"
 else
     AFFINITY_GLOB=""
 fi
 
-if [ ! -d "$WORKSPACE/ompi/mca/pml/ucx" ]; then
-    jenkins_test_ucx="no"
+if [ ! -d "$WORKSPACE/ompi/mca/pml/ucx" ]
```

Comment:
entire `if` can be removed. it was needed during period when not all branches had `ucx` pml and we needed this hack. not anymore.

Reply:
Fixed.